media/js/firefox/feedback-ga-tracking.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
```

Comment:
Is there a reason to use the `document.ready` style here? If not, better to switch to the common IIFE style.